autoupdate-docs-travis.sh

```diff
@@ -8,16 +8,6 @@ set -e
 
 
 if [ "$(git rev-parse master)" = "$TRAVIS_COMMIT" ]; then
```

Comment:
Since it's specified in the circle config that this only runs on master, you can drop this if statement.